pyserini/fusion/_base.py

```diff
@@ -43,5 +49,39 @@ def reciprocal_rank_fusion(runs: List[TrecRun], rrf_k: int = 60, depth: int = No
     """
 
     # TODO: Add option to *not* clone runs, thus making the method destructive, but also more efficient.
-    rrf_runs = [run.clone().rescore(method=FusionMethod.RRF, rrf_k=rrf_k) for run in runs]
-    return TrecRun.merge(rrf_runs, AggregationMethod.SUM, depth, k)
+    rrf_runs = [run.clone().rescore(method=RescoreMethod.RRF, rrf_k=rrf_k) for run in runs]
+    return TrecRun.merge(rrf_runs, AggregationMethod.SUM, depth=depth, k=k)
+
+
+def interpolation(runs: List[TrecRun], alpha: int = 0.5, depth: int = None, k: int = None):
+    """
+    Perform fusion by interpolation on a list of exactly two ``TrecRun`` objects.
+    new_score = first_run_score * alpha + (1 - alpha) * second_run_score.
+
+    Parameters
+    ----------
+    runs : List[TrecRun]
+        List of ``TrecRun`` objects. Exactly two runs.
+    alpha : int
+        Parameters alpha will be applied on the first run and (1 - alpha) will be applied on the second run.
+    depth : int
+        Maximum number of results from each input run to consider. Set to ``None`` by default, which indicates that
+        the complete list of results is considered.
+    k : int
+        Length of final results list.  Set to ``None`` by default, which indicates that the union of all input documents
+        are ranked.
+
+    Returns
+    -------
+    TrecRun
+        Output ``TrecRun`` that combines input runs via interpolation.
+    """
+
+    if len(runs) != 2:
+        raise Exception('Interpolation must be performed on exactly two runs.')
+
+    scaled_runs = []
+    scaled_runs.append(runs[0].clone().rescore(method=RescoreMethod.SCALE, scale=alpha))
```

Comment:
make rescale an option, but `True` by default?

Reply:
Could you please elaborate more? rescore() is a function that takes action on scores depending on the method.

We have RescoreMethod.RRF and Rescore Method.SCALE.